boards/others/esp32c3_supermini/Kconfig.defconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
```

Comment:
This board is not supported by Espressif so I think the copyright message should be updated by the actual vendor.

Reply:
"others" is the vendor name for you.

Reply:
You can use your own name as well:
```
/*
 * Copyright 2024 "Your name"
 *
 * SPDX-License-Identifier: Apache-2.0
 */
 ```